valhalla/userrequests/target_helpers.py

```diff
@@ -42,8 +42,8 @@ def data(self):
 class SiderealTargetHelper(BaseTargetHelper):
     def __init__(self, target):
         self.fields = (
-            'type', 'name', 'ra', 'dec', 'proper_motion_ra', 'proper_motion_dec', 'parallax',
-            'coordinate_system', 'equinox', 'epoch', 'acquire_mode', 'rot_mode', 'rot_angle'
+            'type', 'name', 'ra', 'dec', 'proper_motion_ra', 'proper_motion_dec', 'parallax', 'radvel',
+            'coordinate_system', 'equinox', 'epoch', 'acquire_mode', 'rot_mode', 'rot_angle', 'vmag'
```

Comment:
I'm pretty sure NonSidereal targets have a magnitude as well. In fact all targets do.